From: Debian Javascript Maintainers
 <pkg-javascript-devel@lists.alioth.debian.org>
Date: Sat, 18 Sep 2021 20:25:07 +0000
Subject: remove-babel-preset-airbnb

Replace preset airbnb with @babel/env for babel 7
---
 node-acorn-dynamic-import/.babelrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/node-acorn-dynamic-import/.babelrc
+++ b/node-acorn-dynamic-import/.babelrc
@@ -1,5 +1,5 @@
 {
-  "presets": [["airbnb", {
+  "presets": [["@babel/env", {
     "targets": {
       "node": 8
     }
